examples/vault/vault_examples/encrypt.py

```diff
@@ -9,33 +9,37 @@
 from pangea.utils import str2str_b64
 
 
-def main():
+def main() -> None:
     token = os.getenv("PANGEA_VAULT_TOKEN")
     domain = os.getenv("PANGEA_DOMAIN")
+    assert domain
     config = PangeaConfig(domain=domain)
     vault = Vault(token, config=config)
 
     try:
         # Set a unique name.
-        name = f"Python encrypt example {int(time.time())}"
+        name = f"Python encrypt example {token_hex(8)}"
```

Comment:
Switched these time-based names to crypto-random ones because with the parallel runs we're now doing it was very prone to name collision.